CHANGELOG.md

```diff
@@ -45,6 +45,9 @@ changes.
   + Skip observing history of events before they connected
   + View the transactions in the server output encoded as CBOR
 
+- Replaced existing websocket server with production-grade one. This change also
+  enables serving HTTP API responses if needed.
```

Comment:
Should: not mention http.. any webserver can do that and we are not actively adding this.